### DISCUSSION TITLE
Unable to resolve application issues

### DISCUSSION THREAD

**Discussion by Zviedrys** [community/community#141778] [GitHub Education] [answered]
### Hi everyone,

I have tried applying several times, but I am unable to resolve these two issues.

- The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

In Lithuania our student IDs are issued not by schools or universities but by a government controlled program. The name of the university is present on the card, but as a part of the logo and that is not being recognised.
Additionally, there are English names for the fields on the card, but we are not given a translated one only in English.

- Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

The information matched exactly, but it's not being recognised. I believe it is because of the letter 'Ž', and it's diacritic that is partly intersecting some text above.

Any help or manual review would be greatly appreciated.

**Comment by alexcerezo** [comment, marked as answer]
Hi,

It sounds like you're facing a frustrating situation with the verification process. Here are a couple of suggestions that might help resolve the issues you're encountering:

School Name Not Recognized: Since the university name is part of the logo and is not being recognized, try submitting multiple documents as a workaround. You can provide a combination of your student ID (with the logo), an official transcript, or any other document issued by your university where the full name of your school appears in text. Additionally, including a separate English translation of your student ID, even though it has English fields, might make the recognition process smoother.

Name Mismatch Issue: For the name mismatch due to the letter "Ž" and its diacritic, you might want to try submitting a clearer, higher-resolution image of your document. Ensure the diacritic isn't touching or overlapping with other text. If possible, manually separate the fields by cropping or positioning them in a way that minimizes interference. If that still doesn't work, contacting GitHub support directly and explaining that the diacritic is causing the issue might prompt them to perform a manual review.

These approaches might help in getting your submission accepted. If all else fails, directly reaching out for a manual review is likely your best option.

Good luck, and I hope this gets resolved soon!
